### ISSUE TITLE
Portfolio template Contact me icon in Firefox - Fix it!

### ISSUE THREAD

**Issue by namnaku87** [kaustubhai/portfolio-template#11]
Hello there,

The minor fix was required! When I checked the contact icon of your portfolio template in the Firefox browser, I discovered a glitch in which the portfolio icon was not visible. Please have a check on them fix it. Thanks.

**Comment by kaustubhai** [comment]
Hey what do you mean by `contact icon`? 
Can you share some screenshot to elaborate your issue

**Comment by namnaku87** [comment]
Other browser it's working fine. The icon is not visible on Firefox there must be some bug or maybe not supporting in that browser. 

Check you portfolio website template in Firefox. Icon will not be visible. Need to be fixed.

![Inkedds_LI](https://user-images.githubusercontent.com/90440828/141507862-43690e59-2a19-4dfd-96b2-bc4a541b1d19.jpg)

**Comment by namnaku87** [comment]
Are you there? Did you solve the minor issue?

**Comment by namnaku87** [comment]
Hello, Kaustubhai. It's been a long time since I opened this non-responsive problem. Are you present? Did you resolve the tiny problem?

**Comment by kaustubhai** [comment]
Hello @namnaku87 fix is on the way, RELAX!

**Comment by namnaku87** [comment]
> Hello @namnaku87 fix is on the way, RELAX!

No just reminding you that's it. Okay thanks.

**Comment by namnaku87** [comment]
Is the issue still being fixed, or may I expect it to be resolved by the end of the year?

just wanted to make sure

**Comment by iamazeez** [comment]
@namnaku87  I was looking into your issue as you say its get disappear when you switch to Firefox actually its just go on the bottom of the browser . I think its just a css issue. If I found the bug so I will rise a pr.

**Comment by kaustubhai** [comment]
> Is the issue still being fixed, or may I expect it to be resolved by the end of the year?
> 
> just wanted to make sure

I cant tell you brother exactly, may be end of the this year or next whenever I'll get free time... meanwhile I'll definitely ask you to look into it if you can find the issue and maybe create a PR on your own instead of waiting for someone else to solve it...

**Comment by namnaku87** [comment]
> @namnaku87 I was looking into your issue as you say its get disappear when you switch to Firefox actually its just go on the bottom of the browser . I think its just a css issue. If I found the bug so I will rise a pr.

@iamazeez Yes, if you can try to solve the issue on your own and raise your PR.
